apps/frontend/src/components/inputs/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,52 @@
+import { EyeIcon, EyeSlashIcon } from '@heroicons/react/24/solid';
+import clsx from 'clsx';
+import React, { useState } from 'react';
+
+import type { PasswordInputProps } from './types';
+
+export function PasswordInput({
+  field,
+  form: { touched, errors },
+  label,
+  className,
+  ...props
+}: PasswordInputProps) {
+  const [showPassword, setShowPassword] = useState(false);
+  const toggleShow = () => {
+    setShowPassword((prev) => !prev);
+  };
+
+  return (
+    <div>
+      {label ? <span className="label p-0">{label}</span> : null}
+      <div className="relative border-green w-full mt-2">
+        <input
+          type={showPassword ? 'text' : 'password'}
```

Comment:
By introducing password show/hide functionality we no longer need "Repeat password" fields across the platform. Please remove them.